vertexai/token-count/tokencount_test.go

```diff
@@ -26,19 +26,18 @@ import (
 func Test_countTokens(t *testing.T) {
 	tc := testutil.SystemTest(t)
 
-	prompt := "why is the sky blue?"
 	location := "us-central1"
 	modelName := "gemini-1.5-flash-001"
 
 	var buf bytes.Buffer
-	err := countTokens(&buf, prompt, tc.ProjectID, location, modelName)
+	err := countTokens(&buf, tc.ProjectID, location, modelName)
 	if err != nil {
 		t.Fatalf("Test_countTokens: %v", err.Error())
 	}
 
 	answer := buf.String()
 	s := strings.TrimPrefix(answer, "Number of tokens for the prompt: ")
-	s = strings.TrimSpace(s)
+	s = strings.Split(s, "\n")[0]
```

Comment:
suggestion: please add test for the second print added in the sample ("Total number of tokens for the prompt...")